thirdparty/cJSON/cJSON.c

```diff
@@ -550,7 +550,7 @@ static cJSON_bool print_number(const cJSON * const item, printbuffer * const out
 static unsigned parse_hex4(const unsigned char * const input)
 {
     unsigned int h = 0;
-    size_t i = 0;
+    size_t i;
 
```

Comment:
Code in thirdparty/ should not be modified, except to fix obvious bugs, to reduce the chance of conflicts when updating it with newer versions from the upstream sources.
This is not one of these cases.